src/engines/kotor/gui/guibackground.h

```diff
@@ -0,0 +1,86 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  The menu background.
+ */
+
+#ifndef ENGINES_KOTOR_GUI_GUIBACKGROUND_H
+#define ENGINES_KOTOR_GUI_GUIBACKGROUND_H
+
+#include "src/graphics/guielement.h"
+
+#include "src/graphics/aurora/texturehandle.h"
+
+#include "src/events/notifyable.h"
+
+namespace Common {
+	class UString;
+}
+
+namespace Engines {
+
+namespace KotOR {
+
+static const Common::UString &kBackgroundTypeMenu   = "back";
+static const Common::UString &kBackgroundTypeComp0  = "comp0";
+static const Common::UString &kBackgroundTypeComp1  = "comp1";
+static const Common::UString &kBackgroundTypeLoad   = "load";
+static const Common::UString &kBackgroundTypeMap    = "map";
+static const Common::UString &kBackgroundTypePazaak = "pazaak";
+static const Common::UString &kBackgroundTypeStore  = "store";
+
+class GuiBackground : public Graphics::GUIElement, public Events::Notifyable {
```

Comment:
Is there a reason you used "Gui" instead of "GUI"? I think everywhere else I spelled it "GUI".